app/components/works/contributor_row_component.rb

```diff
@@ -4,11 +4,17 @@
 module Works
   # Renders a widget corresponding to a single contributor to the work.
   class ContributorRowComponent < ApplicationComponent
-    sig { params(form: ActionView::Helpers::FormBuilder, required: T::Boolean, citation: T::Boolean).void }
-    def initialize(form:, required: false, citation: false)
+    sig do
+      params(form: ActionView::Helpers::FormBuilder,
+             required: T::Boolean,
+             citation: T::Boolean,
+             optional: T::Boolean).void
+    end
+    def initialize(form:, required: false, citation: false, optional: false)
       @form = form
       @required = required
       @citation = citation
+      @optional = optional
```

Comment:
Isn't optional implied when required is false?

Reply:
Doh, yes. I totally missed that. Will update.